liblangutil/Scanner.cpp

```diff
@@ -271,12 +274,60 @@ bool Scanner::skipWhitespaceExceptUnicodeLinebreak()
 	return sourcePos() != startPosition;
 }
 
+
+/// Tries to scan for an RLO/LRO/RLE/LRE/PDF and keeps track of script writing direction override depth.
+///
+/// @returns ScannerError::NoError in case of successful parsing and directional encodings are paired
+///          and error code in case the input's lexical parser state is invalid and this error should be reported
+///          to the user.
+static ScannerError parseUnicodeDirectionDepth(shared_ptr<CharStream> const& _stream, size_t _startPosition)
+{
+	static array<pair<string_view, int>, 5> constexpr m_sequences{
```

Comment:
It's not a member variable so it should not have the `m_` prefix. BTW, do we actually have any convention for constants?
```suggestion
	static array<pair<string_view, int>, 5> constexpr sequences{
```

Also, I don't think it needs to be `static` if it's a constant.

Reply:
Convention is `c_`, but I would say it is optional.

Reply:
Thanks. The only thing I saw mentioned in the coding style are static public constants:

> static const variables that form an external API.

Reply:
Renamed to `directionalSequences` to be more descriptive